src/editor/Core/Filesystem.re

```diff
@@ -255,13 +255,21 @@ let copy = (source, dest) =>
   );
 
 let createOniConfiguration = (~configDir, ~file) => {
-  open Utility;
+  let userConfigPath = Utility.join([configDir, file]);
```

Comment:
I can't remember - do we create the `configDir` somewhere? Or do we need to check if that folder exists, and create it here?